apps/docs/src/pages/colors.mdx

```diff
@@ -0,0 +1,231 @@
+---
+title: Colors
+description: Sample Colors documentation
+layout: ../layouts/MainLayout.astro
+---
+
+import { ColorWrapper, ColorsWrapper } from '@selleo/core/src/Colors';
+import { H2 } from '@selleo/core/src/Headers';
+import Preview from '../components/Preview.astro';
+import Description from '../components/Description.astro';
+
+<Description>
+  Below you can find Colors components styled with tailwindcss.
+</Description>
+
+<H2>Brand Colors</H2>
+
+<Preview>
+  <ColorWrapper
```

Comment:
Those things shouldn't be used in with Preview, since those are not components we'd like to use, just a showcase

Reply:
Okay, if I would know that it's only for a showcase, I would implement it different way 😄, can I keep these components as they are? I don't think changing the ColorWrapper into plane code in that file would be beneficial

Reply:
Okay